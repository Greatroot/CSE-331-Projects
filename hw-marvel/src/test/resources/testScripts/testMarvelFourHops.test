# Tests finding the four hop path from THANDEMOS to URCHALLA.

LoadGraph marvel marvel.tsv

FindPath marvel THANADEMOS URCHALLA