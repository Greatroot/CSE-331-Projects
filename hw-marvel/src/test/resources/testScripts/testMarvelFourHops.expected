# Tests finding the four hop path from THANDEMOS to URCHALLA.

loaded graph marvel

path from THANADEMOS to URCHALLA:
THANADEMOS to IRON FIST/DANIEL RAN via M/CP 111/2
IRON FIST/DANIEL RAN to ABOMINATION/EMIL BLO via H2 278
ABOMINATION/EMIL BLO to X-MAN/NATHAN GREY via XM 32
X-MAN/NATHAN GREY to URCHALLA via XM 61